shared/reducers/chat.js

```diff
@@ -281,9 +281,67 @@ function reducer(state: Types.State = initialState, action: ChatGen.Actions) {
       const {payload: {teamJoinSuccess}} = action
       return state.set('teamJoinSuccess', teamJoinSuccess)
     }
-  }
+    // Saga only actions
+    case ChatGen.updateBadging:
+    case ChatGen.attachmentLoaded:
+    case ChatGen.attachmentSaveFailed:
+    case ChatGen.attachmentSaveStart:
+    case ChatGen.attachmentSaved:
+    case ChatGen.badgeAppForChat:
+    case ChatGen.blockConversation:
+    case ChatGen.deleteMessage:
+    case ChatGen.downloadProgress:
+    case ChatGen.editMessage:
+    case ChatGen.getInboxAndUnbox:
+    case ChatGen.inboxStale:
+    case ChatGen.inboxStoreLoaded:
+    case ChatGen.incomingMessage:
+    case ChatGen.incomingTyping:
+    case ChatGen.joinConversation:
+    case ChatGen.leaveConversation:
+    case ChatGen.loadAttachment:
+    case ChatGen.loadAttachmentPreview:
+    case ChatGen.loadInbox:
+    case ChatGen.loadMoreMessages:
+    case ChatGen.markSeenMessage:
+    case ChatGen.muteConversation:
+    case ChatGen.openAttachmentPopup:
+    case ChatGen.openConversation:
+    case ChatGen.openFolder:
+    case ChatGen.openTeamConversation:
+    case ChatGen.openTlfInChat:
+    case ChatGen.outboxMessageBecameReal:
+    case ChatGen.postMessage:
+    case ChatGen.removeOutboxMessage:
+    case ChatGen.retryAttachment:
+    case ChatGen.retryMessage:
+    case ChatGen.saveAttachment:
+    case ChatGen.saveAttachmentNative:
+    case ChatGen.selectAttachment:
+    case ChatGen.selectConversation:
+    case ChatGen.selectNext:
+    case ChatGen.setNotifications:
+    case ChatGen.setupChatHandlers:
+    case ChatGen.shareAttachment:
+    case ChatGen.startConversation:
+    case ChatGen.toggleChannelWideNotifications:
+    case ChatGen.unboxConversations:
+    case ChatGen.unboxMore:
+    case ChatGen.updateInboxComplete:
+    case ChatGen.updateMetadata:
+    case ChatGen.updateSnippet:
+    case ChatGen.updateTempMessage:
+    case ChatGen.updateThread:
+    case ChatGen.updateTyping:
+    case ChatGen.updatedNotifications:
+    case ChatGen.uploadProgress:
+      return state
 
-  return state
+    default:
+      // eslint-disable-next-line no-unused-expressions
+      (action: empty) // if you get a flow error here it means there's an action you claim to handle but didn't
```

Comment:
annoyingly flow will complain but won't necessarily tell you which one is missing